veros/setup/acc_basic/acc_basic.py

```diff
@@ -0,0 +1,162 @@
+        vs.eke_k_max = 1e4
+        vs.eke_c_k = 0.4
+        vs.eke_c_eps = 0.5
+        vs.eke_cross = 2.
+        vs.eke_crhin = 1.0
+        vs.eke_lmin = 100.0
+        vs.enable_eke_superbee_advection = False
+        vs.enable_eke_isopycnal_diffusion = False
+
+        vs.enable_idemix = False
+        vs.enable_idemix_hor_diffusion = False
+        vs.enable_eke_diss_surfbot = False
+        vs.eke_diss_surfbot_frac = 0.2
+        vs.enable_idemix_superbee_advection = False
+
+        vs.eq_of_state_type = 3
+
+    @veros_method
+    def set_grid(self, vs):
+        ddz = np.array([50., 70., 100., 140., 190., 240., 290., 340.,
+                        390., 440., 490., 540., 590., 640., 690.])
+        vs.dxt[...] = 2.0
+        vs.dyt[...] = 2.0
+        vs.x_origin = 0.0
+        vs.y_origin = -40.0
+        vs.dzt[...] = ddz[::-1] / 2.5
+
+    @veros_method
+    def set_coriolis(self, vs):
+        vs.coriolis_t[:, :] = 2 * vs.omega * np.sin(vs.yt[None, :] / 180. * vs.pi)
+
+    @veros_method
+    def set_topography(self, vs):
+        x, y = np.meshgrid(vs.xt, vs.yt, indexing='ij')
+        vs.kbot[...] = np.logical_or(x > 1.0, y < -20).astype(np.int)
+
+    @veros_method
+    def set_initial_conditions(self, vs):
+        # initial conditions
+        vs.temp[:, :, :, 0:2] = ((1 - vs.zt[None, None, :] / vs.zw[0]) * 15 * vs.maskT)[..., None]
+        vs.salt[:, :, :, 0:2] = 35.0 * vs.maskT[..., None]
+
+        # wind stress forcing
+        yt_min = global_min(vs, vs.yt.min())
+        yu_min = global_min(vs, vs.yu.min())
+        yt_max = global_max(vs, vs.yt.max())
+        yu_max = global_max(vs, vs.yu.max())
+
+        taux = allocate(vs, ('yt',))
+        taux[vs.yt < -20] = 0.1 * np.sin(vs.pi * (vs.yu[vs.yt < -20] - yu_min) / (-20.0 - yt_min))
+        taux[vs.yt > 10] = 0.1 * (1 - np.cos(2 * vs.pi * (vs.yu[vs.yt > 10] - 10.0) / (yu_max - 10.0)))
+        vs.surface_taux[:, :] = taux * vs.maskU[:, :, -1]
+
+        # surface heatflux forcing
+        vs._t_star = allocate(vs, ('yt',), fill=15)
+        vs._t_star[vs.yt < -20] = 15 * (vs.yt[vs.yt < -20] - yt_min) / (-20 - yt_min)
+        vs._t_star[vs.yt > 20] = 15 * (1 - (vs.yt[vs.yt > 20] - 20) / (yt_max - 20))
+        vs._t_rest = vs.dzt[None, -1] / (30. * 86400.) * vs.maskT[:, :, -1]
+
+        if vs.enable_tke:
+            vs.forc_tke_surface[2:-2, 2:-2] = np.sqrt((0.5 * (vs.surface_taux[2:-2, 2:-2] + vs.surface_taux[1:-3, 2:-2]) / vs.rho_0)**2
+                                                      + (0.5 * (vs.surface_tauy[2:-2, 2:-2] + vs.surface_tauy[2:-2, 1:-3]) / vs.rho_0)**2)**(1.5)
+
+        if vs.enable_idemix:
+            vs.forc_iw_bottom[...] = 1e-6 * vs.maskW[:, :, -1]
+            vs.forc_iw_surface[...] = 1e-7 * vs.maskW[:, :, -1]
+
+    @veros_method
+    def set_forcing(self, vs):
+        vs.forc_temp_surface[...] = vs._t_rest * (vs._t_star - vs.temp[:, :, -1, vs.tau])
+
+    @veros_method
+    def set_diagnostics(self, vs):
```

Comment:
I think the spec here was "no snapshots"

Reply:
Exactly! So I removed snapshots + Markus wanted me to remove energy output as well.
Do you want me to formalise their absence somehow?

Reply:
Oh, you did? I thought snapshots are active by default, but maybe I remember it wrong.

Reply:
I ran the model for a test and neither snapshots nor energy output files were created.